kolibri/plugins/coach/api.py

```diff
@@ -80,5 +85,14 @@ class RecentReportViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         query_node = ContentNode.objects.get(pk=self.kwargs['content_node_id'])
-        recent_content_items = ContentSummaryLog.objects.filter_by_topic(query_node).order_by('end_timestamp').values_list('content_id')
+        if self.request.query_params.get('last_active_time'):
+            # Last active time specified
+            datetime_cutoff = parse(self.request.query_params.get('last_active_time'))
+        else:
+            datetime_cutoff = timezone.now() - datetime.timedelta(7)
```

Comment:
Ah cool, so if I don't pass it, it'll as for 7 days of reports by default? What was it you were saying about the servers not being reliable with their time logs?